src/pages/dmt5-trading-signals/_signal.tsx

```diff
@@ -104,27 +106,40 @@ const StyledChecklist = styled(Checklist)`
     }
 `
 
-type SignalProps = {
-    content: {
-        header: React.ReactElement
-        text: React.ReactElement
-        list: React.ReactElement[]
+export type SignalContent = {
+    header: {
+        text: TString
+        components?: ReactElement[]
     }
+    text: TString
+    list: TString[]
+}
+
+type SignalProps = {
+    content: SignalContent
 }
 
 export const Signal = ({ content }: SignalProps) => {
+    const { header, text, list }: SignalContent = content
+
     return (
         <StyledSection background="var(--color-white)">
             <Flex direction="column" max_width="99.6rem" m="0 auto" jc="space-between" ai="center">
-                <StyledText>{content.text}</StyledText>
+                <StyledText>
+                    <Localize translate_text={text} />
+                </StyledText>
                 <ContentWrapper>
-                    <StyledHeader>{content.header}</StyledHeader>
+                    <StyledHeader>
+                        <Localize translate_text={header.text} components={header.components} />
+                    </StyledHeader>
                     <CardWrapper>
-                        {content.list.map((text, idx) => {
+                        {list.map((text, idx) => {
                             return (
                                 <Card key={idx}>
```

Comment:
`text`in unique here right? we can replace idx on text
```suggestion
                        {list.map((tex) => {
                            return (
                                <Card key={tex}>
```